src/main/java/com/logentries/log4j/LogentriesAppender.java

```diff
@@ -152,22 +152,41 @@ protected void append( LoggingEvent event) {
 		// Append stack trace if present and layout does not handle it
 		if (layout.ignoresThrowable()) {
 			String[] stack = event.getThrowableStrRep();
-			if (stack != null)
-			{
-				int len = stack.length;
-				formattedEvent += ", ";
-				for(int i = 0; i < len; i++)
-				{
-					formattedEvent += stack[i];
-					if(i < len - 1)
-						formattedEvent += "\u2028";
-				}
-			}
+			if (stack != null && stack.length > 0)
+				formattedEvent = appendFormatedEvent(formattedEvent, stack);
 		}
 				
 		// Prepare to be queued
 		this.le_async.addLineToQueue(formattedEvent);
 	}
+	private static final String EXCEPTION_SEPARATOR = ", ";
+	
+	static String appendFormatedEvent(final String formattedEvent, final String[] stack) {
+		final int len = stack.length;
+		
+		// calculate buffer size
+		int buffSize = formattedEvent.length();
+		buffSize += EXCEPTION_SEPARATOR.length();
+		for (int i = 0; i < len; i++) {
+			buffSize += stack[i].length();
+			if (i < len - 1) {
+				buffSize += 1;
+			}
+		}
+		
```

Comment:
There is something that might saved more cpu cycles here.
Replacing: 
```
int buffSize = formattedEvent.length();
buffSize += EXCEPTION_SEPARATOR.length();
```
By:
```
int buffSize = formattedEvent.length() + EXCEPTION_SEPARATOR.length();
```
And removing:
```
if (i < len - 1) {
	buffSize += 1;
}
```
And adding after for loop:
```
buffSize += len - 1;
```
It would save...